admin_guide/cluster_management.rst

```diff
@@ -226,6 +226,17 @@ Citus places table rows into worker shards based on the hashed value of the rows
 
 However sharing shards can cause resource contention when tenants differ drastically in size. This is a common situation for systems with a large number of tenants -- we have observed that the size of tenant data tend to follow a Zipfian distribution as the number of tenants increases. This means there are a few very large tenants, and many smaller ones. To improve resource allocation and make guarantees of tenant QoS it is worthwhile to move large tenants to dedicated nodes.
 
+The :ref:`citus_stat_statements <citus_stat_statements>` view can help pinpoint resource usage by tenant. Here's how to see which tenant runs most queries:
+
+.. code-block:: sql
+
+  select partition_key, sum(calls) as total_queries
```

Comment:
@mtuncer I removed the time stuff from this query. Is there anything I could add to make the query more useful, or does this look good?

Reply:
It might be nice to show that this view could be joined with pg_stat_statements for extra information. (like. max/min/avg/total_time). But we should somehow word it such that those values are not this particular tenant specific, but for all tenants running this query.